js/main.js

```diff
@@ -19,9 +20,57 @@ $entryForm.addEventListener('submit', function (event) {
   data.entries.unshift(formInputValue);
   $photoPreview.src = './images/placeholder-image-square.jpg';
   $entryForm.reset();
+  var list = renderEntryDomTree(formInputValue);
+  entryHolder.prepend(list);
+  entryHolder.scrollIntoView();
 });
 
 window.addEventListener('beforeunload', function () {
   var formStorage = JSON.stringify(data);
   localStorage.setItem('code-journal-input', formStorage);
 });
+
+function renderEntryDomTree(entry) {
+  var descriptionEntry = document.createElement('p');
+  var descriptionText = document.createTextNode(entry.notes);
+  descriptionEntry.appendChild(descriptionText);
+
+  var headingEntry = document.createElement('h2');
+  var headingText = document.createTextNode(entry.title);
+  headingEntry.appendChild(headingText);
+
+  var divInputSection = document.createElement('div');
+  divInputSection.setAttribute('class', 'column-half input-section');
+
+  divInputSection.append(headingEntry, descriptionEntry);
+
+  var imgEntry = document.createElement('img');
+  imgEntry.src = entry.photoURL;
+
+  var divImageSection = document.createElement('div');
+  divImageSection.setAttribute('class', 'column-half image-section');
+
+  divImageSection.append(imgEntry);
+
+  var liAnEntry = document.createElement('li');
+  liAnEntry.setAttribute('class', 'row an-entry');
+
+  liAnEntry.append(divImageSection, divInputSection);
+
+  return liAnEntry;
+}
+window.addEventListener('DOMContentLoaded', function () {
+  var storageLocal = localStorage.getItem('code-journal-input');
+  if (storageLocal !== null) {
+    var storageData = JSON.parse(storageLocal);
+    data.view = storageData.view;
+    data.entries = storageData.entries;
+    data.editing = storageData.editing;
+    data.nextEntryId = storageData.nextEntryId;
+  }
```

Comment:
LocalStorage stuff should all be being done in `data.js`. I recommend you review `javascript-local-storage`, the code will be nearly identical